### DISCUSSION TITLE
Grid Component Set Maximum Height

### DISCUSSION THREAD

**Discussion by joselitogatchalianalonzo** [vikramlearning/blazorbootstrap#249] [Q&A] [answered]
How can I set the maximum height of the Grid? Please note that I set AllowPaging="false". I'm using Grid on my Dashboard together with other items. However, it is taking the full height of the screen leaving no space for other items. How can I set the maximum height of the Grid without paging i.e. AllowPaging="false"?

**Comment by gvreddy04** [comment]
@joselitogatchalianalonzo You can use the below custom CSS appraoch.

Keep the grid inside a div and give a CSS class name. In the blow example, CSS class name is `employees`.

```cshtml
<div class="employees">
    <Grid @ref="grid"
          TItem="Employee4"
          Class="table table-hover table-bordered table-striped"
          DataProvider="EmployeesDataProvider"
          AllowFiltering="true"
          Responsive="true">

          ....

    </Grid>
</div>
```

Add the below CSS in the .css file

```css
div.employees > div.table-responsive {
    height: 200px !important;
}
```

**Screenshot:**

![image](https://github.com/vikramlearning/blazorbootstrap/assets/2337067/4b1a8039-af3f-41c6-83e8-2f0ba8c0e858)

**Comment by gvreddy04** [comment, marked as answer]
@joselitogatchalianalonzo In the future version, we will add **Height** paramter to the Blazor Grid component

**Comment by joselitogatchalianalonzo** [comment]
Thanks @gvreddy04. I will try your sample code.
